lib/requester/core.js

```diff
@@ -67,7 +68,14 @@ var _ = require('lodash'),
 
         // removes the `referer` header when a redirect happens (default: false)
         // @note `referer` header set in the initial request will be preserved during redirect chain
-        removeRefererHeader: 'removeRefererHeaderOnRedirect'
+        removeRefererHeader: 'removeRefererHeaderOnRedirect',
+
+        // Controls whether or not request should be sent with cache-control header, if true request will contain
+        // 'cache-control: no-cache' header
+        noCacheHeader: 'noCacheHeader',
```

Comment:
Why are these added to protocol profile behaviors?

Reply:
Coz you would want to override the behaviour for maybe a few individual requests and not for the entire collection ?